projects/plugins/jetpack/class.jetpack-user-agent.php

```diff
@@ -5,660 +5,7 @@
-	 */
-	public static function get_blackbeberry_OS_version() {
-		_deprecated_function( __METHOD__, 'Jetpack 8.7', '\Automattic\Jetpack\Device_Detection\User_Agent_Info::get_blackbeberry_OS_version from the `automattic/jetpack-device-detection` package' );
-		return User_Agent_Info::get_blackbeberry_OS_version();
-	}
-
-	/**
-	 * Retrieve the blackberry browser version.
-	 *
-	 * Return string are from the following list:
-	 * - blackberry-10
-	 * - blackberry-webkit
-	 * - blackberry-5
-	 * - blackberry-4.7
-	 * - blackberry-4.6
-	 *
-	 * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
-	 *
-	 * @return string Type of the BB browser. If browser's version is not found, detect_blackbeberry_browser_version will return boolean false.
-	 */
-	public static function detect_blackberry_browser_version() {
-		_deprecated_function( __METHOD__, 'Jetpack 8.7', '\Automattic\Jetpack\Device_Detection\User_Agent_Info::detect_blackberry_browser_version from the `automattic/jetpack-device-detection` package' );
-		return User_Agent_Info::detect_blackberry_browser_version();
-	}
-
-	/**
-	 * Checks if a visitor is coming from one of the WordPress mobile apps.
-	 *
-	 * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
-	 *
-	 * @return bool
-	 */
-	public static function is_mobile_app() {
-		_deprecated_function( __METHOD__, 'Jetpack 8.7', '\Automattic\Jetpack\Device_Detection\User_Agent_Info::is_mobile_app from the `automattic/jetpack-device-detection` package' );
-		return User_Agent_Info::is_mobile_app();
-	}
-
-	/**
-	 * Detects if the current browser is Nintendo 3DS handheld.
-	 *
-	 * Example: Mozilla/5.0 (Nintendo 3DS; U; ; en) Version/1.7498.US
-	 * can differ in language, version and region
-	 *
-	 * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
-	 */
-	public static function is_Nintendo_3DS() {
-		_deprecated_function( __METHOD__, 'Jetpack 8.7', '\Automattic\Jetpack\Device_Detection\User_Agent_Info::is_Nintendo_3DS from the `automattic/jetpack-device-detection` package' );
-		return User_Agent_Info::is_Nintendo_3DS();
-	}
-
-	/**
-	 * Was the current request made by a known bot?
-	 *
-	 * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
-	 *
-	 * @return boolean
-	 */
-	public static function is_bot() {
-		_deprecated_function( __METHOD__, 'Jetpack 8.7', '\Automattic\Jetpack\Device_Detection\User_Agent_Info::is_bot from the `automattic/jetpack-device-detection` package' );
-		return User_Agent_Info::is_bot();
-	}
 
-	/**
-	 * Is the given user-agent a known bot?
-	 * If you want an is_bot check for the current request's UA, use is_bot() instead of passing a user-agent to this method.
-	 *
-	 * @param string $ua A user-agent string.
-	 *
-	 * @deprecated 8.7.0 Use Automattic\Jetpack\Device_Detection\User_Agent_Info
-	 *
-	 * @return boolean
-	 */
-	public static function is_bot_user_agent( $ua = null ) {
```

Comment:
This function is used in the following w.org plugins:

- [ ] `jetpack-omnisearch`: omnisearch/class.jetpack-user-agent.php,1376
- [ ] `jp-custom-css`: custom-css/class.jetpack-user-agent.php,1376
- [ ] `themedy-toolbox`: custom-css/custom-css/class.jetpack-user-agent.php,1367
- [ ] `wpvkp-ultimate-shortcodes`: custom-css/custom-css/class.jetpack-user-agent.php,1367

Reply:
All of these instances are fine. They are bringing in the class (though with issues as noted above).